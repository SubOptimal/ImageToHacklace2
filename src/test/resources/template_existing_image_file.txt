HL
\00 7A 01\ It's hip to be [[square.gif]] ! \00\
\FF\

