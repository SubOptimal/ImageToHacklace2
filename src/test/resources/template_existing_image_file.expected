HL
\00 7A 01\ It's hip to be \1F 08 00 7E 42 42 42 42 7E 00\ ! \00\
\FF\

